### PR TITLE
Add CDI collection scripts

### PR DESCRIPTION
**Pull request by awels**
Added CDI namespace and collection script. 

Signed-off-by: Alexander Wels <awels@redhat.com>

### CODE REVIEW

File: gather_cnv_cdi [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    do
+        echo "POD: $pod"
+        pod_collection_path=$ns_collection_path/$pod
+        mkdir -p $pod_collection_path
+
+        /usr/bin/openshift-must-gather inspect -n $ns pod/$pod
+        oc logs $pod -n $ns > $pod_collection_path/$pod.log
+    done
+
+    for pvc in $(oc get pvc -n $ns --no-headers -o=custom-columns=NAME:.metadata.name 2>/dev/null)
```
Comment (masayag) [inline gather_cnv_cdi]: Is there a need to collect also PVs or storage classes ?

Comment (djfjeff) [reply gather_cnv_cdi]: The base must-gather tool already collect PVs and storage class. I don't think we need to duplicate them here.

Comment (awels) [reply gather_cnv_cdi]: Exactly why I didn't grab those here, also PV and storage class are cluster level resources while PVCs are namespaced.

Comment (masayag) [reply gather_cnv_cdi]: Thank you both, I should have look first at https://github.com/openshift/must-gather/blob/master/collection-scripts/gather#L22

**Pull request merged**